scripts/talos/README.md

```diff
@@ -0,0 +1,53 @@
+# Talos
+
+## Description
+
+Talos is a bumper bot that runs automatically for Psammead packages.
+
+## Manual Usage
+
```

Comment:
An extra explanation here would be useful.
~~~
Manual usage of Talos requires access to our Jenkins CI pipeline. So for now it is limited to those employed by the BBC. If you're an external contributor and want Talos run manually, please feel free to [open an issue](https://github.com/bbc/psammead/issues/new/choose).